specs/core/0_beacon-chain.md

```diff
@@ -1463,14 +1464,16 @@ def get_changed_validators(validators: List[ValidatorRecord],
             )
         if validators[i].status == EXITED_WITHOUT_PENALTY:
             validators[i].latest_status_change_slot = current_slot
-            total_changed += get_effective_balance(validators[i])
+            total_exited_balance += get_effective_balance(validators[i])
             validator_registry_delta_chain_tip = get_new_validator_registry_delta_chain_tip(
                 validator_registry_delta_chain_tip=validator_registry_delta_chain_tip,
                 index=i,
                 pubkey=validators[i].pubkey,
                 flag=EXIT,
             )
-        if total_changed >= max_allowable_change:
+        if total_activated_balance >= max_allowable_balance_changes:
```

Comment:
This doesn't quite do what we're looking for. 

I think it should be 
```python
if validators[i].status == PENDING_ACTIVATION and total_activated_balance < max_allowable_balance_changes:
    ...
if validators[i].status == EXITED_WITHOUT_PENALTY and total_exited_balance < max_allowable_balance_changes:
    ...
if total_activated_balance >= max_allowable_balance_changes and total_exited_balance >= max_allowable_balance_changes:
    break

Reply:
I think both don't quite do it :) What about now?